src/services/SolidService.ts

```diff
@@ -81,7 +81,9 @@ class SolidService {
   ): Promise<any> => {
     const fetchSessionData = session.fetch;
     let myCovidFile: any;
-    const info = [date, certificaat, validationDate];
+
+    const d = new Date(date);
+    const v = new Date(validationDate);
```

Comment:
Avoid using single letter variables, it's best to give these full meaningful camelcased names